airflow-core/src/airflow/ui/src/i18n/locales/pl/dag.json

```
@@ -66,6 +66,19 @@
       "label": "Kierunek grafu"
     }
   },
+  "paramsFailed": "Nie udało się załadować parametrów",
+  "parse": {
+    "toaster": {
+      "error": {
+        "description": "Nie udało się przetworzyć żądania parsowania Daga. Mogą istnieć oczekujące żądania parsowania do przetworzenia.",
+        "title": "Nie udało się ponownie sparsować Daga"
+      },
+      "success": {
+        "description": "Dag zostanie wkrótce ponownie sparsowany.",
+        "title": "Żądanie ponownego parsowania zostało pomyślnie wysłane"
```

```suggestion
        "description": "Dag zostanie wkrótce ponownie przetworzony.",
        "title": "Żądanie ponownego przetworzenia zostało pomyślnie wysłane"
```
I think we used `przetworzyć` for `parse`